src/app/Caches/Create/Features/Backups/BackupsSite.tsx

```diff
@@ -0,0 +1,282 @@
+                                    {t('caches.create.configurations.feature.mode-auto')}
+                                </Text>
+                            </TextContent>
+                        }
+                    />
+                </FormGroup>
+            </React.Fragment>
+        )
+    }
+
+
+    // Options for Failure Policy
+    const failurePolicyOptions = () => {
+        return Object.keys(BackupSiteFailurePolicy).map((key) => (
+            <SelectOption key={key} value={BackupSiteFailurePolicy[key]} />
+        ));
+    };
+
+    return (
+        <Form>
+            <Flex>
+                <FlexItem grow={{ default: 'grow' }} style={{ maxWidth: '25rem' }}>
+                    <FormGroup
+                        fieldId='strategy'
+                        isInline
+                    >
+                        <MoreInfoTooltip label={t('caches.create.configurations.feature.strategy')} toolTip={t('caches.create.configurations.feature.strategy-tooltip')} textComponent={TextVariants.h2} />
+                        <Radio
+                            name="strategy-radio"
+                            id="async"
+                            onChange={() => setStrategy(BackupSiteStrategy.ASYNC)}
+                            isChecked={strategy as BackupSiteStrategy == BackupSiteStrategy.ASYNC}
+                            label={
+                                <TextContent>
+                                    <Text component={TextVariants.h3}>
+                                        {t('caches.create.configurations.feature.strategy-async')}
+                                    </Text>
+                                </TextContent>
+                            }
+                        />
+                        <Radio
+                            name="strategy-radio"
+                            id="sync"
+                            onChange={() => setStrategy(BackupSiteStrategy.SYNC)}
+                            isChecked={strategy as BackupSiteStrategy == BackupSiteStrategy.SYNC}
+                            label={
+                                <TextContent>
+                                    <Text component={TextVariants.h3}>
+                                        {t('caches.create.configurations.feature.strategy-sync')}
+                                    </Text>
+                                </TextContent>
+                            }
+                        />
+                    </FormGroup>
+                </FlexItem>
+                <FlexItem grow={{ default: 'grow' }} style={{ maxWidth: '25rem' }}>
+                    <FormGroup
+                        fieldId='twoPhaseCommit'
+                        isInline
+                    >
+                        <MoreInfoTooltip label={t('caches.create.configurations.feature.two-phase-commit')} toolTip={t('caches.create.configurations.feature.two-phase-commit-tooltip')} textComponent={TextVariants.h2} />
```

Comment:
Two-phase commit is for SYNC backups only but I can still select this if I change the strategy to ASYNC.